internal/service/quicksight/dashboard_test.go

```diff
@@ -156,6 +156,8 @@ func TestAccQuickSightDashboard_updateVersionNumber(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "name", rNameUpdated),
 					resource.TestCheckResourceAttr(resourceName, "status", quicksight.ResourceStatusCreationSuccessful),
 					resource.TestCheckResourceAttr(resourceName, "version_number", "2"),
+					testAccCheckDashboardVersionExists(ctx, resourceName, 1, &dashboard),
+					testAccCheckDashboardName(&dashboard, rName),
```

Comment:
For clarity, these should indicate that they're referencing version 1, e.g.

```suggestion
					testAccCheckDashboardVersionExists(ctx, resourceName, 1, &dashboardV1),
					testAccCheckDashboardName(&dashboardV1, rName),
```

Reply:
Updated to `dashboardV1`